pennylane/grouping/graph_colouring.py

```diff
@@ -47,18 +47,15 @@ def largest_first(binary_observables, adj):
 
     **Example**
 
-    >>> binary_observables
-    array([[1., 1., 0.],
-           [1., 0., 0.],
-           [0., 0., 1.],
-           [1., 0., 1.]])
-    >>> adj
-    array([[0., 0., 1.],
-           [0., 0., 1.],
-           [1., 1., 0.]])
+    >>> binary_observables = np.array([[1., 1., 0.],
+                                       [1., 0., 0.],
+                                       [0., 0., 1.],
+                                       [1., 0., 1.]])
+    >>> adj = np.array([[0., 0., 1.],
+                        [0., 0., 1.],
+                        [1., 1., 0.]])
```

Comment:
```suggestion
    >>> binary_observables = np.array([[1., 1., 0.],
    ... [1., 0., 0.],
    ... [0., 0., 1.],
    ... [1., 0., 1.]])
    >>> adj = np.array([[0., 0., 1.],
    ... [0., 0., 1.],
    ... [1., 1., 0.]])
```
Just a slight change to ensure that it conforms to pycon syntax